site/content/docs/developer-guide/plugin-manifest.md

```diff
@@ -95,8 +95,8 @@ accessible to download from a user’s machine. The relevant fields are:
 ## Specifying platform-specific instructions
 
 Krew makes it possible to install the same plugin on different operating systems
-(e.g., Windows, macOS, and Linux) and different architectures
-(e.g., `amd64`, `386`, and `arm`).
+(e.g., `windows`, `darwin` (macOS), and `linux`) and different architectures
```

Comment:
```suggestion
(e.g., `windows`, `darwin`/macOS, and `linux`) and different architectures
```
Nested parentheses in text look wonky.

Reply:
hmm I agree, I think the suggestion also looks a little wonky as well though. this is how it was before though with the nested parentheses. @ahmetb @corneliusweig do either of you have a preference?